### DISCUSSION TITLE
pool plotting question

### DISCUSSION THREAD

**Discussion by 1buckeyefan** [swar/Swar-Chia-Plot-Manager#1225] [Q&A]
I have the updated files to enable pool plotting and I've added the pool_url to my config with the contract address. When I try to plot new pools they are not getting assigned the contract address but rather creating stand alone plots. How do I get the contract address attached to new plots?

**Comment by Chilli71** [comment]
17 days ago. Wow. Was pool plotting already released by chia at that time?
Could you fix it with the latest version of plotmanager or do you still need help?
